PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutInternalClient.java

```diff
@@ -60,7 +60,7 @@ void sendRequest(final Context context, final PayPalNativeRequest payPalRequest,
                                         String pairingIdKey = isBillingAgreement ? "ba_token" : "token";
                                         String pairingId = parsedRedirectUri.getQueryParameter(pairingIdKey);
                                         String clientMetadataId = payPalRequest.getRiskCorrelationId() != null
-                                                ? payPalRequest.getRiskCorrelationId() : payPalDataCollector.getClientMetadataId(context, configuration, payPalRequest.hasUserLocationConsent());
+                                                ? payPalRequest.getRiskCorrelationId() : payPalDataCollector.getClientMetadataId(context, configuration, false);
```

Comment:
Should this be `true` for this version to maintain existing behavior? Since we are saying the fix for this module will be in the next version?

Reply:
Or that might unnecessarily flag compliant apps, so could go either way

Reply:
Hm yeah, I could go either way. Maybe keeping the existing behavior makes the most sense (passing true).

Reply:
Actually second guessing myself again - I think let's keep it false